assets/base/i18n/en.json

```diff
@@ -3202,7 +3207,7 @@
   "user.settings.notifications.email.disabled": "Email notifications are not enabled",
   "user.settings.notifications.email.disabled_long": "Email notifications have not been enabled by your System Administrator.",
   "user.settings.notifications.email.everyHour": "Every hour",
-  "user.settings.notifications.email.everyXMinutes": "Every {count, plural, one {minute} other {{count, number} minutes}}",
+  "user.settings.notifications.email.fifteenMinutes": "Every 15 minutes",
```

Comment:
I think where just using this for fifteen minutes.  Hence, fix it to that value.

Reply:
This is going to affect the web app as well. Do we need to change this or can we just leave it as is?

Reply:
I left it as is and just added `user.settings.notifications.email.fifteenMinutes`.